### DISCUSSION TITLE
部署后能正常打开realtek audio console么

### DISCUSSION THREAD

**Discussion by x51d6g4** [meetrevision/playbook#127] [Q&A]
其他系统打开realtek audio console闪退

**Comment by HarryVasanth** [comment]
> [!TIP]
> - 首先，建议您今后尽量使用英语提问，这样可以方便社区成员和维护者更好地理解并为您提供帮助。
> - First of all, I kindly suggest writing your questions in English in the future, as it makes it much easier for everyone to understand and assist you

Yes, you should be able to open and use the Realtek Audio Console normally after deploying the ReviOS Playbook. The playbook does not intentionally remove or break the services required for this application to function.

> [!NOTE]  
> ReviOS disables automatic driver installation via Windows Update by default. If you experience crashes or the application closes unexpectedly, it is highly likely related to missing or incorrect audio drivers. Please manually download and install the latest Realtek audio drivers and the corresponding Audio Console application from your motherboard or laptop manufacturer's official website.
